From d3afc63097489c186f7b90066914b830f888851a Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Thu, 1 Apr 2021 13:23:23 -0400
Subject: [PATCH] Drop -Werror

---
 mypyc/build.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mypyc/build.py b/mypyc/build.py
index 088e481fc..e16b3d69d 100644
--- a/mypyc/build.py
+++ b/mypyc/build.py
@@ -506,7 +506,7 @@ def mypycify(
     cflags = []  # type: List[str]
     if compiler.compiler_type == 'unix':
         cflags += [
-            '-O{}'.format(opt_level), '-Werror', '-Wno-unused-function', '-Wno-unused-label',
+            '-O{}'.format(opt_level), '-Wno-unused-function', '-Wno-unused-label',
             '-Wno-unreachable-code', '-Wno-unused-variable',
             '-Wno-unused-command-line-argument', '-Wno-unknown-warning-option',
         ]
-- 
2.30.2

